docs/styles/extra.css

```
@@ -9,3 +14,100 @@
+@media screen and (min-width: 1220px) {
+  .show-on-small {
+    display: none;
+  }
+}
+
+.move-down {
+  transform: translate(0px, -5px);
+  animation: down_movement 1000ms infinite ease-out;
+}
+
+@keyframes down_movement {
```

Stylelint (17.14.0)

[error] Expected keyframe name "down_movement" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)